src/editor/ui_menus/main_menu_map_options.cc

```diff
@@ -394,10 +394,17 @@ MainMenuMapOptions::MainMenuMapOptions(EditorInteractive& parent, Registry& regi
 
 	tags_box_.add_space(labelh_);
 
-	tags_box_.add(new UI::Textarea(&tags_box_, UI::PanelStyle::kWui, UI::FontStyle::kWuiLabel, 0, 0,
-	                               max_w_, labelh_, _("Waterway length limit:")));
+	UI::Textarea* ww_text =
+	   new UI::Textarea(&tags_box_, UI::PanelStyle::kWui, UI::FontStyle::kWuiLabel, 0, 0, max_w_,
+	                    labelh_, _("Ferry range:"));
+	std::string ww_tooltip = _("Enable ferries, waterways, and ferry yards on this map by setting"
+	                           " maximum length of waterways for ferries.");
```

Comment:
Fine :-)

But shouldn't it be "… by setting **a** maximum length of waterways for ferries."?